pkg/api/plugin.go

```diff
@@ -121,4 +123,7 @@ type GenevaActions interface {
 
 	// GetControlPlanePods fetches a consolidated list of the control plane pods in the cluster
 	GetControlPlanePods(ctx context.Context, oc *OpenShiftManagedCluster) ([]byte, error)
+
+	// BackupEtcdCluster backs up the cluster's etcd
+	BackupEtcdCluster(ctx context.Context, cs *OpenShiftManagedCluster, backupName string) *PluginError
```

Comment:
can return error instead of *PluginError here as well

Reply:
:+1: